pages/careers/_open-positions.html

```diff
@@ -1,8 +1,28 @@
 <div class="row row-page-header">
   <div class="col-xs-12">
     <h2>Open Positions</h2>
-    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're looking for:</p>
+    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're currently looking for:</p>
     <h3>Senior Full-Stack Engineer</h3>
+    <h4>What You'll Do</h4>
+    <ul>
+      <li><strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on Serverless apps in Node, write Terraform modules, write Python scripts and Bash scripts, do frontend development, learn the newest tech from AWS, invent new ways of managing infrastructure. We're looking for true full-stack engineers who can go deep in any area. We're less concerned with prior experience than we are with demonstrated ability to go deep when necessary.</li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>Demonstrated experience building good software.</li>
+      <li>Bonus points for visible open source work.</li>
+      <li>Bonus points for any technical presentations you've done in the past.</li>
+      <li>Bonus points for a blog.</li>
+      <li>Bonus points for something in your profile that makes us smile.</li>
```

Comment:
What do you see as the benefit of listing these?

Reply:
These are things we actually value in a candidate. My thought is that by making it more clear what we value, people are better able to either self-identify as a good fit for present what we're looking to see.

Reply:
I cleaned these up a bit. 8bc6233bef92310eb9cf0b7fee1f219c5bf17875